packages/e2e-test-utils-playwright/src/editor/click-block-appender.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Clicks the default block appender.
+ *
+ * @param {Editor} this
+ */
+export async function clickBlockAppender( this: Editor ) {
+	// The block appender is only visible when there's no selection.
+	await this.page.evaluate( () =>
+		// @ts-ignore (Reason: wp isn't typed)
+		window.wp.data.dispatch( 'core/block-editor' ).clearSelectedBlock()
+	);
+	const appender = await this.page.waitForSelector(
+		'.block-editor-default-block-appender__content'
```

Comment:
You should be able to use a role selector for this, this is what my dev tools say:
![Screen Shot 2022-08-10 at 1 19 27 pm](https://user-images.githubusercontent.com/677833/183821888-d110488a-835d-4b11-a798-b49ddb4f9105.png)